superset/models/slice.py

```diff
@@ -59,7 +59,7 @@
     metadata,
     Column("id", Integer, primary_key=True),
     Column("user_id", Integer, ForeignKey("ab_user.id")),
-    Column("slice_id", Integer, ForeignKey("slices.id")),
+    Column("slice_id", Integer, ForeignKey("slices.id", ondelete="CASCADE")),
```

Comment:
`ondelete="CASCADE"` will not work with sqllite (unless FK support is enabled). Won't emptying owners be enough?

Reply:
Ah, good point... yeah, emptying the owners should be enough. Let me remove the `ondelete`.